### PR TITLE
[SR-4893] Complex column type(array...) is not supported for StarRock…

### PR DESCRIPTION
**Pull request by wuleistarrocks**

### CODE REVIEW

**Comment by wuleistarrocks** [comment]
ref #478

**Comment by wuleistarrocks** [comment]
#477

**Review by chaoyli — APPROVED** [review]

File: gensrc/thrift/FrontendService.thrift [hunk 1]
```diff
@@ -729,8 +729,20 @@ struct TIndexInfo {
...
+  5: optional i32 columnScale
+  6: optional string columnKey
+  7: optional bool key
+  8: optional string aggregationType
+  9: optional string comment
+}
+
 struct TSchemaMeta {
-    1: optional list<TColumnDef> columns
+    1: optional list<TColumnMeta> columns
```
Comment (imay) [inline gensrc/thrift/FrontendService.thrift:745]: Is this used for any others? This change will cause compatibility issues

Comment (wuleistarrocks) [reply gensrc/thrift/FrontendService.thrift:745]: no, TSchemaMeta is just used in get table meta result

**Review by imay — APPROVED** [review]

**Comment by wuleistarrocks** [comment]
Fix issue #477

**Pull request merged**